mkosi/__init__.py

```diff
@@ -1460,6 +1460,21 @@ def build_kernel_modules_initrd(context: Context, kver: str) -> Path:
     return kmods
 
 
+def find_requested_devicetree(context: Context, kver: str) -> Path:
+    assert context.config.devicetree
+
+    for f in (
```

Comment:
Let's call this `d` or `p`, since it's a path/directory, but definitely not a file